components/n-ui/subscription-offer-prompt/lionel.js

```diff
@@ -74,12 +74,56 @@ const createPopupHTML = values =>
 
 
 const createSubscriptionPrompt = values => {
+	let focusedElementBeforePrompt;
+	let focusableElementsStrings = ['.subscription-prompt--subscribe-btn', '.n-sliding-popup-close'];
+
 	const subscriptionPrompt = createPopupHTML(values);
-	subscriptionPrompt.onClose = setPromptLastClosed;
+	let focusableElements = subscriptionPrompt.querySelectorAll(focusableElementsStrings);
+	focusableElements = Array.prototype.slice.call(focusableElements);
+
+	subscriptionPrompt.onClose = () => {
+		setPromptLastClosed();
+
+		if(focusedElementBeforePrompt !== undefined)	focusedElementBeforePrompt.focus();
```

Comment:
Ideally we'd have `focusedElementBeforePrompt.focus();` in a new line, enclosed in `{}`s even if it's one line, to keep with Next standards(ish) - My bad, I didn't mention that before! 🙈